### PR TITLE
Add getting started links

### PR DESCRIPTION
**Pull request by mark-wiemer**
A good central resource for some in-depth guides that will be updated by the larger community, not just Bedrock OSS.

I know there is a wiki guide but that can get outdated quick and referencing external and official guides will keep our resources high-quality.

The wiki guides, I believe, are quite intimidating in their multi-section structure. The Microsoft Learn guides are all one big page, which is generally less scary.

### CODE REVIEW

File: docs/meta/useful-links.md [hunk 1]
```diff
@@ -77,7 +77,18 @@ Important links have a ⭐.
...
+- [Build a gameplay experience with TypeScript](https://learn.microsoft.com/en-us/minecraft/creator/documents/scriptinggettingstarted): TypeScript is Microsoft's copy of JavaScript, it's popular and becoming more popular! Writing mods in TypeScript allows you to add any functionality you can imagine to your mod!
+- [mojang-minecraft Module](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/mojang-minecraft/mojang-minecraft): This module and the others near it are how we can access Minecraft values with our TypeScript code. It's technical, but a great resource.
+- [List and summary of commands (Minecraft fandom)](https://minecraft.fandom.com/wiki/Commands#List_and_summary_of_commands): Most mods will run some commands. This community-supported wiki is the best resource for learning each and every command.
+- [Bedrock OSS](https://github.com/Bedrock-OSS): Bedrock OSS is a great centralized community around building Minecraft mods. The Discord is popular, the wiki is awesome, and the supported projects are top-notch. And it's getting better all the time!
```
Comment (mark-wiemer) [inline docs/meta/useful-links.md:89]:
```suggestion
```

File: docs/meta/useful-links.md [hunk 2]
```diff
@@ -77,7 +77,17 @@ Important links have a ⭐.
...
+- [Introduction to the GameTest Framework](https://learn.microsoft.com/en-us/minecraft/creator/documents/gametestgettingstarted): This is the best way to test games, and it uses JavaScript, the most popular programming language in the world!
+- [Build a gameplay experience with TypeScript](https://learn.microsoft.com/en-us/minecraft/creator/documents/scriptinggettingstarted): TypeScript is Microsoft's copy of JavaScript, it's popular and becoming more popular! Writing mods in TypeScript allows you to add any functionality you can imagine to your mod!
+- [mojang-minecraft Module](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/mojang-minecraft/mojang-minecraft): This module and the others near it are how we can access Minecraft values with our TypeScript code. It's technical, but a great resource.
+- [List and summary of commands (Minecraft fandom)](https://minecraft.fandom.com/wiki/Commands#List_and_summary_of_commands): Most mods will run some commands. This community-supported wiki is the best resource for learning each and every command.
+
+## Sample ehavior & Resource Packs
```
Comment (mark-wiemer) [inline docs/meta/useful-links.md:90]:
```suggestion
## Sample behavior & Resource Packs
```

File: docs/meta/useful-links.md [hunk 3]
```diff
@@ -77,7 +77,17 @@ Important links have a ⭐.
...
+- [Getting Started with Add-On Development for Bedrock Edition](https://learn.microsoft.com/en-us/minecraft/creator/documents/gettingstarted): These guides show you exactly how to build your first resource pack and your first behavior pack from start to finish.
+- [Molang: a Beginner's Guide](https://learn.microsoft.com/en-us/minecraft/creator/documents/molangbeginnersguide): Molang is a Minecraft programming language that can be useful for writing some advanced add-ons.
+- [Introduction to the GameTest Framework](https://learn.microsoft.com/en-us/minecraft/creator/documents/gametestgettingstarted): This is the best way to test games, and it uses JavaScript, the most popular programming language in the world!
+- [Build a gameplay experience with TypeScript](https://learn.microsoft.com/en-us/minecraft/creator/documents/scriptinggettingstarted): TypeScript is Microsoft's copy of JavaScript, it's popular and becoming more popular! Writing mods in TypeScript allows you to add any functionality you can imagine to your mod!
```
Comment (mark-wiemer) [inline docs/meta/useful-links.md:86]:
```suggestion
- [Build a gameplay experience with TypeScript](https://learn.microsoft.com/en-us/minecraft/creator/documents/scriptinggettingstarted): TypeScript is Microsoft's copy of JavaScript. Writing mods in TypeScript allows you to add any functionality you can imagine to your mod!
```

File: docs/meta/useful-links.md [hunk 4]
```diff
@@ -77,7 +77,17 @@ Important links have a ⭐.
 -   [Bedrock Texture Pack Template](https://github.com/Brennian/BedrockTexturesTemplate)
 -   [Documentation Graveyard (removed components)](https://gist.github.com/destruc7i0n/ea1a6a7f97f0986d9326c58246f96fa3)
 
-## Behavior & Resource Packs
+### Getting Started With Your First Add-On
+
+- ⭐ [Minecraft Creator Portal](https://learn.microsoft.com/en-us/minecraft/creator/): This is the official Microsoft-supported main page for everything creators need to know to get started making resource packs and behavior packs.
```
Comment (mark-wiemer) [inline docs/meta/useful-links.md:82]:
```suggestion
```

**Comment by SmokeyStack** [comment]
Should change all instances of `mod` to `add-on` as that's the official name

File: docs/meta/useful-links.md [hunk 5]
```diff
@@ -77,7 +77,16 @@ Important links have a ⭐.
...
 -   [Documentation Graveyard (removed components)](https://gist.github.com/destruc7i0n/ea1a6a7f97f0986d9326c58246f96fa3)
 
-## Behavior & Resource Packs
+### Getting Started With Your First Add-On
+
+- [Getting Started with Add-On Development for Bedrock Edition](https://learn.microsoft.com/en-us/minecraft/creator/documents/gettingstarted): These guides show you exactly how to build your first resource pack and your first behavior pack from start to finish.
+- [Molang: a Beginner's Guide](https://learn.microsoft.com/en-us/minecraft/creator/documents/molangbeginnersguide): Molang is a Minecraft programming language that can be useful for writing some advanced add-ons.
+- [Introduction to the GameTest Framework](https://learn.microsoft.com/en-us/minecraft/creator/documents/gametestgettingstarted): This is the best way to test games, and it uses JavaScript, the most popular programming language in the world!
+- [Build a gameplay experience with TypeScript](https://learn.microsoft.com/en-us/minecraft/creator/documents/scriptinggettingstarted): TypeScript is Microsoft's copy of JavaScript. Writing mods in TypeScript allows you to add any functionality you can imagine to your mod!
```
Comment (mark-wiemer) [inline docs/meta/useful-links.md:85]:
```suggestion
- [Build a gameplay experience with TypeScript](https://learn.microsoft.com/en-us/minecraft/creator/documents/scriptinggettingstarted): TypeScript is Microsoft's copy of JavaScript. Writing add-ons in TypeScript allows you to add any functionality you can imagine to your mod!
```

File: docs/meta/useful-links.md [hunk 6]
```diff
@@ -77,7 +77,16 @@ Important links have a ⭐.
...
+- [mojang-minecraft Module](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/mojang-minecraft/mojang-minecraft): This module and the others near it are how we can access Minecraft values with our TypeScript code. It's technical, but a great resource.
+- [List and summary of commands (Minecraft fandom)](https://minecraft.fandom.com/wiki/Commands#List_and_summary_of_commands): Most mods will run some commands. This community-supported wiki is the best resource for learning each and every command.
```
Comment (mark-wiemer) [inline docs/meta/useful-links.md:87]:
```suggestion
- [List and summary of commands (Minecraft fandom)](https://minecraft.fandom.com/wiki/Commands#List_and_summary_of_commands): Most add-ons will run some commands. This community-supported wiki is the best resource for learning each and every command.
```

File: docs/meta/useful-links.md [hunk 7]
```diff
@@ -77,7 +77,16 @@ Important links have a ⭐.
...
 -   [Documentation Graveyard (removed components)](https://gist.github.com/destruc7i0n/ea1a6a7f97f0986d9326c58246f96fa3)
 
-## Behavior & Resource Packs
+### Getting Started With Your First Add-On
+
+- [Getting Started with Add-On Development for Bedrock Edition](https://learn.microsoft.com/en-us/minecraft/creator/documents/gettingstarted): These guides show you exactly how to build your first resource pack and your first behavior pack from start to finish.
+- [Molang: a Beginner's Guide](https://learn.microsoft.com/en-us/minecraft/creator/documents/molangbeginnersguide): Molang is a Minecraft programming language that can be useful for writing some advanced add-ons.
+- [Introduction to the GameTest Framework](https://learn.microsoft.com/en-us/minecraft/creator/documents/gametestgettingstarted): This is the best way to test games, and it uses JavaScript, the most popular programming language in the world!
+- [Build a gameplay experience with TypeScript](https://learn.microsoft.com/en-us/minecraft/creator/documents/scriptinggettingstarted): TypeScript is Microsoft's copy of JavaScript. Writing add-ons in TypeScript allows you to add any functionality you can imagine to your mod!
```
Comment (mark-wiemer) [inline docs/meta/useful-links.md:85]:
```suggestion
- [Build a gameplay experience with TypeScript](https://learn.microsoft.com/en-us/minecraft/creator/documents/scriptinggettingstarted): TypeScript is Microsoft's copy of JavaScript. Writing add-ons in TypeScript allows you to add any functionality you can imagine!
```

**Comment by mark-wiemer** [comment]
> Should change all instances of `mod` to `add-on` as that's the official name

Thanks @SmokeyStack , I've updated the text

**Pull request merged**